lib/src/objector.dart

```diff
@@ -93,11 +93,15 @@ class Objector extends RedditBase {
       return buildModeratorAction(reddit, data['data']);
     } else if (data.containsKey('sr') &&
         data.containsKey('comment_karma') &&
-        data.containsKey('link_karma')) {
+        data.containsKey('link_karma') &&
```

Comment:
This conditional seems to be for a `Subreddit` but awarder/awardee karma are `Redditor` properties. I think you can just drop the changes in this file and things should work as-is.

Reply:
Seems like i didn't pay attention to that at first. It should be resolved in the newest commit.